### PR TITLE
just-index

### PR DESCRIPTION
**Pull request by ecto**
my first contribution - let me know if i can make it better!

### CODE REVIEW

File: packages/array-index/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  index([], null); // undefined
+  index({}, 'id'); // undefined
+*/
+
+function index(arr, key) {
+  if (!Array.isArray(arr)) {
+    return undefined;
+  }
+
+  if (!key || typeof key !== 'string') {
```
Comment (angus-c) [inline packages/array-index/index.js]: if (typeof key !== 'string') {

File: packages/array-index/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "just-index",
+  "version": "1.0.0",
+  "description": "returns an object keyed from array of objects",
```
Comment (angus-c) [inline packages/array-index/package.json]: could be clearer maybe? "return an object from an array, keyed by the value at the given id".

File: packages/array-index/index.js (hunk 1 above)
Comment (ecto) [inline packages/array-index/index.js]: good catch!

File: packages/array-index/package.json (hunk 2 above)
Comment (ecto) [inline packages/array-index/package.json]: changed!

**Comment by ecto** [comment]
thanks for entertaining my idea! i haven't done any open source work for awhile and i've missed contributing :)

**Pull request merged**